docs/adal-connectivity.md

```diff
@@ -0,0 +1,57 @@
+# adal-connectivity
+Android library to verify if the device has an Internet connection and what it's type is!
```

Comment:
@paribeiro10 replacing "and what it's type is!" with "and what type it is" seems more clear to read.

Reply:
Duly noted. Replaced.